src/main/java/seedu/address/model/UserPrefs.java

```diff
@@ -14,8 +14,8 @@
 public class UserPrefs implements ReadOnlyUserPrefs {
 
     private GuiSettings guiSettings = new GuiSettings();
-    private String name = "Annon";
-    private Path foodDiaryFilePath = Paths.get("data" , "fooddiary.json");
+    private String name = "anonymous";
+    private Path foodDiaryFilePath = Paths.get("data" , "foodDiary.json");
```

Comment:
Is there a reason for the change in the Json file path? The capitalization in the new string might result in a missing file.

Reply:
Done